product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -100,17 +100,17 @@ __OUTPUT__
 ```
 
 This command runs the `vacuum full foo` command on the node-one and node-three nodes.
-Note that the node names are passed to the function in an array.
+The node names are passed to the function in an array.
```

Comment:
Again we are drawing attention to this.

Reply:
"Note that" is classic doc wordiness and doesn't really add any meaning. Do you want to somehow be more elaborate like, "as opposed to blah blah, in this case the node names are..."